readme.md

```diff
@@ -65,6 +65,13 @@ Number of digits to appear after the milliseconds decimal point.
 
 Useful in combination with [`process.hrtime()`](https://nodejs.org/api/process.html#process_process_hrtime).
 
+##### keepDecimalsOnWholeSeconds
+
+Type: `boolean`<br>
+Default: `false`
+
+Keep milliseconds on whole seconds: `13s` → `13.0s`.
```

Comment:
Can you include a use-case? Users will see this option and wonder when they would want to use it.

Reply:
Sure. Just updated it to contain an example.